its/autoscan/src/test/resources/autoscan/diffs/diff_S1130.json

```diff
@@ -1,6 +1,6 @@
 {
   "ruleKey": "S1130",
   "hasTruePositives": true,
-  "falseNegatives": 26,
+  "falseNegatives": 25,
```

Comment:
This change is because AutoScanTest analyzes not only source examples in `java-checks-test-sources` but also a real project: `java-checks-test-sources/test-classpath-reader`, and this file has been changed: `java-checks-test-sources/test-classpath-reader/src/test/java/org/sonar/java/test/classpath/TestClasspathUtilsTest.java`